mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -63,6 +65,7 @@
     private final Field[] idFields;
     private final Path[] idPaths;
     private final String[] mongoIdFields;
+    private final String txid=ObjectId.get().toString();
```

Comment:
Renamed txid to docver.

On Tue, Aug 9, 2016 at 12:29 PM, Alec Henninger <notifications@github.com>
wrote:

> In mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/299#discussion_r74117217>
> :
>
> > @@ -63,6 +65,7 @@
> >      private final Field[] idFields;
> >      private final Path[] idPaths;
> >      private final String[] mongoIdFields;
> > +    private final String txid=ObjectId.get().toString();
>
> Some places use txid, some use docver (docversion,etc). I think maybe all
> of them should just be called docVer/docVersion to be clear they are the
> same thing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/299/files/3215df526dc8cf645462d5147d4ecfe0db012c92#r74117217>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADgDDfTjHSS4rPJaJp2Kh-llccb-gD3nks5qeMcGgaJpZM4JfOdd>
> .
>